backend/app/services/message_forwarding_service.py

```
@@ -0,0 +1,290 @@
+        note: Optional[str],
+        priority: QueueMessagePriority,
+    ) -> None:
+        """Forward messages to all members of a group."""
+        from app.models.namespace_member import NamespaceMember
+
+        # Get all group members
+        members = (
+            db.query(NamespaceMember)
+            .filter(
+                NamespaceMember.namespace == group_namespace,
+                NamespaceMember.is_active == True,
```

Ruff (0.15.6)

[error] Avoid equality comparisons to `True`; use `NamespaceMember.is_active:` for truth checks

Replace with `NamespaceMember.is_active`

(E712)